pyaedt/modeler/Object3d.py

```diff
@@ -1,13 +1,6 @@
-# -*- coding: utf-8 -*-
```

Comment:
Thanks for removing this.

As a note, python files are utf-8 by default, so this comment isn't necessary.

Reply:
It has to be there. Ironpython will fail otherwise